monitoring.adoc

```diff
@@ -834,10 +843,6 @@ on October 25, 2017, 28 minutes after midnight, a POP3 client with IP address `2
 
 image:images/poplogmessage.png[Mailbox Log Entry for POP]
```

Comment:
The above section I would move into Log pattern. I thinks it gives a great visual example of that specific section.